config/routes.rb

```diff
@@ -1,16 +1,19 @@
 Rails.application.routes.draw do
   root 'flights#index'
   get 'flights' => 'flights#index'
+  get 'pay_ticket' => 'bookings#pay_for_ticket'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.